### PR TITLE
Added template for forms

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+9. Insert the Google Tag Manager script into your website and head over to your website to fill the form.
+
+10. All responses captured will be stored in the forms.io application under the response tab.
+
+
+### Support
+Contact our support team through “Live Chat” or “Help us improve” options through our application.
+
+
+![alt text](https://infinity.500apps.com/img/pushninja/GTM-Image-1.png)
```
Comment (subramaniyamkolluri) [inline README.md]: change the images to forms.io

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+If your website integrates Google Tag Manager, adding forms to your site doesn't require any coding. All you'll have to do is add a new tag based on the Forms.io community template published by Google.
+
+1. Log in to Google Tag Manager, navigate your account and click on "Add a new tag".
+
+2. Search for our community template "Forms.io by 500apps" in the "Community Template Gallery".
+
+3. Add the template to your tag by clicking on "Add to Workspace".
+
+4. Copy the key from the application and place it in the "Forms.io Key" field in your tag after the template is added.
```
Comment (subramaniyamkolluri) [inline README.md]: We might need DIV-ID? too

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: added images for forms

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: changed  the readme

File: README.md [hunk 3]
```diff
@@ -15,30 +15,30 @@ If your website integrates Google Tag Manager, adding forms to your site doesn't
 
 3. Add the template to your tag by clicking on "Add to Workspace".
 
-4. Copy the key from the application and place it in the "Forms.io Key" field in your tag after the template is added.
+4. First copy the dev-id (<div id='DIV-ID'></div>) and place it in the ”Forms.io Key”
```
Comment (subramaniyamkolluri) [inline README.md]: What is Dev-id

Comment (ghost) [reply README.md]: changed readme file

**Pull request merged**